DesignSafe-CI/portal
# Hotfix: use new system monitor

## Overview: ##

DS was using the old system monitor, causing issues with specific systems. This removes some cases where the form would be disabled, and provides a less scary message if the system is down.

## PR Status: ##


## Testing Steps: ##
1. Go to https://designsafe.dev/rw/workspace/#!/OpenSees::Simulation and select the Stampede2 version of the app
2. Confirm no error message appears
3. Select the Frontera version of the Opensees app
4. Confirm no error message appears
